citi-onboarding/Maracatec
# Feature/front navbar banner

### What I Did
Create button, navbar e banner1 components
### How to Test

1. Open the terminal and go to the folder of this project

2. In terminal:

 i. `git checkout develop`
ii. `git pull`
iii. `git checkout feature/FrontNavbarBanner`
iv. `git pull origin feature/FrontNavbarBanner`
v. `open the folder server with comand cd client/`
vi. `yarn install or npm install`
vii. `yarn start or npm start`

 3. Check if the component looks just like the mockup in different screen sizes

4. Read the changed files section on GitHub

5. Test the things you think are worth testing, even the ones that are not in this description
